modules/installation-configuration-parameters.adoc

```diff
@@ -174,3 +181,27 @@ For more information about AWS tags,
 see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources]
 in the AWS documentation.
 |====
+
+
+.Additional Azure parameters
+[cols=".^2,.^3a,.^3a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`machines.platform.azure.type`
+|The Azure VM instance type.
+|Any valid VM type.
```

Comment:
@jhixson74, re: https://github.com/openshift/installer/pull/2254#issuecomment-527235540, if we have limits on the types of VMs that are acceptable, I want to list them here.

Reply:
Azure can only run Windows & Linux. The specific versions are listed here: https://docs.microsoft.com/en-us/azure-stack/operator/azure-stack-supported-os?view=azs-1908

Reply:
I'm adding that link.